src/components/Charts/ChartFutureHospitalization.tsx

```diff
@@ -117,7 +118,7 @@ const ChartFutureHospitalization = ({
 
   const renderTooltip = (p: PointProjections) => (
     <Tooltip left={marginLeft + getXCoord(p)} top={marginTop + getYCoord(p)}>
-      {getTooltipBody(p)}
+      <TooltipStyle.BodyMuted>{getTooltipBody(p)}</TooltipStyle.BodyMuted>
```

Comment:
It seems weird to me that we're using `BodyMuted` instead of `Body` for all of the non-rt charts.  Is that because they get too big, otherwise?  Can we not just make the tooltip bigger or something?

Reply:
Updated - I think we need to make the tooltip for hospitalizations more consistent with the others so we don't need to tweak it here.